src/styles/header.scss

```diff
@@ -0,0 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+}
+
+.menu__link {
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  position: relative;
+  height: 100%;
+  display: inline-block;
+  line-height: 60px;
+}
+
+.menu__link:hover {
+  color: var(--primary-color);
+}
```

Comment:
On hover, you're changing the color of the `.menu__link`. However, according to the task, you should also add an underline to the hovered link. You can achieve this by adding `text-decoration: underline;` in the `.menu__link:hover` class.